mon-pix/app/templates/components/routes/login-or-register.hbs

```diff
@@ -0,0 +1,32 @@
+<div class="login-or-register">
+  <div class="panel login-or-register__panel">
+    <div>
+      <img src="/images/pix-logo.svg" alt="Pix" class="login-or-register-panel__logo">
+    </div>
+    <div class="login-or-register-panel__forms-container">
+      <div class="login-or-register-panel__form">
+        <h1 class="form-title">Je m’inscris sur Pix</h1>
+        {{#unless displayRegisterForm}}
+          <button id="register" {{action "toggleFormsVisibility"}} class="button button--white button--bordered button--uppercase">S'inscrire</button>
+        {{/unless}}
+        <div class={{concat "login-or-register-panel-form__expandable" (if displayRegisterForm " login-or-register-panel-form__expandable--expanded")}}>
```

Comment:
Je crois qu'on peut faire comme ceci ici : 
```js
<div class="login-or-register-panel-form__expandable 
     {{#if displayRegisterForm}}login-or-register-panel-form__expandable--expanded{{/if}}">
```
(https://stackoverflow.com/questions/41465461/add-class-to-element-with-handlebars-based-on-property)